platforms/ide/problems-api/src/main/java/org/gradle/api/problems/deprecation/DeprecateMethodSpec.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2025 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.api.problems.deprecation;
+
+import org.gradle.api.Incubating;
+
+/**
+ * Specification for building deprecations related to methods.
+ *
+ * @since 8.14
+ */
+@Incubating
+public interface DeprecateMethodSpec extends DeprecateSpec {
```

Comment:
:x: Same. What's the point of this empty interface? IIUC we can introduce them later without breaking compatibility.